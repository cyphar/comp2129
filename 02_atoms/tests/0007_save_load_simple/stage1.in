LOAD stage0.as
PLAYFROM END
DISPLAY
UNDO
DISPLAY
