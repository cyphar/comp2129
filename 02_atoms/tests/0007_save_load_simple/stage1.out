Game Loaded

Game Ready
Red's Turn


+--------+
|R1|  |  |
|  |G1|  |
|  |  |  |
+--------+

Green's Turn


+--------+
|R1|  |  |
|  |  |  |
|  |  |  |
+--------+

